app/views/admin/facility_groups/_form.html.erb

```diff
@@ -26,19 +26,14 @@
                         disabled: action_name == "edit" ? true : false %>
       <% end %>
 
-      <% if facility_group.facilities.present? %>
-        <%= form.collection_check_boxes :facility_ids, facility_group.facilities, :id, :name, label: "Associated facilities", checked_value: 1 %>
-      <% end %>
-
-      <% if Facility.where(facility_group: nil).present? %>
-        <%= form.collection_check_boxes :facility_ids, Facility.where(facility_group: nil), :id, :name, label: "Unassociated facilities", checked: true %>
-      <% else %>
-        <% if facility_group.new_record? %>
-          <div class="row">
-            <div class="col-md-9 mb-3 text-warning">All existing facilities belong to a facility group.</div>
-          </div>
+      <label>Associated Facilities</label>
```

Comment:
Is it indeed useful to have the associated facilities here? I wanna make sure we're being mindful of [what we learned recently](https://simpledotorg.slack.com/archives/CK95QFJA2/p1605017237290200)

Reply:
Maybe? I also want to make minimal changes here since the broader goal is making sure only good data remains in the system. Keeping this around makes sure there's not much radical change to this UI.

Reply:
Spatially yes, but conceptually it's a bigger change I think. In any case, I'll let you close the loop with Claudio on it. Not a blocker for this PR.

Reply:
BTW this label can go inside the conditional below.

Reply:
> BTW this label can go inside the conditional below.

> Update: Or we can put a "None" in gray, like the "No data to show" pattern in the reports.

Oh, good point.

> Spatially yes, but conceptually it's a bigger change I think. In any case, I'll let you close the loop with Claudio on it. Not a blocker for this PR.

Yeah, I think the idea is that we need to re-think our Region UI (including this page). This is probably the wrong time to implement (not so much to start thinking about the design) because Regions are in a very weird intermediate state.

For now, I'm mostly going by Daniel's approval of removing [this](https://simpledotorg.slack.com/archives/GJ2AUUASK/p1605075341001500).

Reply:
Removing, approved by Daniel. HAHA.